AltServer/AppDelegate.swift

```diff
@@ -67,14 +67,15 @@ class AppDelegate: NSObject, NSApplicationDelegate {
         ALTDeviceManager.shared.start()
         
         let item = NSStatusBar.system.statusItem(withLength: -1)
-        guard let button = item.button else { return }
-        
-        button.image = NSImage(named: "MenuBarIcon")
-        button.target = self
-        button.action = #selector(AppDelegate.presentMenu)
-        
+        item.menu = self.appMenu
+        item.button?.image = NSImage(named: "MenuBarIcon")
+        if let button = item.button
+        {
+            button.image = NSImage(named: "MenuBarIcon")
+        }
```

Comment:
Redundant image assignment? I agree early return doesn't make as much sense here, so I'd prefer just the optional chaining.
```suggestion
        item.button?.image = NSImage(named: "MenuBarIcon") 
```

Reply:
Oops, definitely didn't mean to assign the image twice